packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -342,7 +343,7 @@ class HeapTreeViewState extends State<HeapTree> with AutoDisposeMixin {
         const SizedBox(width: defaultSpacing),
         Row(
           children: [
-            const Text('Heat Map'),
+            const Text('Treemap'),
```

Comment:
use camel case: Treemap --> TreeMap. Here and elsewhere

Reply:
Oh okay. I think either is fine as long as we are consistent. Leaving as is is good.